pkg/ui/src/views/shared/components/sortabletable/index.tsx

```diff
@@ -128,16 +155,49 @@ export class SortableTable extends React.Component<TableProps, {}> {
             "sort-table__row",
             "sort-table__row--body",
             this.props.rowClass(rowIndex),
+            { "sort-table__row--expandable": !!expandableConfig },
           );
-          return (
-            <tr key={rowIndex} className={classes}>
+          const expanded = expandableConfig && expandableConfig.rowIsExpanded(rowIndex);
+          const output = [
+            <tr
+              key={rowIndex}
+              className={classes}
+              onClick={() => expandableConfig.onChangeExpansion(rowIndex, !expanded)}
+            >
+              {expandableConfig ? this.expansionControl(expanded) : null}
               {
                 _.map(columns, (c: SortableColumn, colIndex: number) => {
-                  return <td className={classNames("sort-table__cell", c.className)} key={colIndex}>{c.cell(rowIndex)}</td>;
+                  return (
+                    <td className={classNames("sort-table__cell", c.className)} key={colIndex}>
+                      {c.cell(rowIndex)}
+                    </td>
+                  );
                 })
               }
-            </tr>
-          );
+            </tr>,
+          ];
+          if (expandableConfig && expandableConfig.rowIsExpanded(rowIndex)) {
+            const expandedAreaClasses = classNames(
+              "sort-table__row",
+              "sort-table__row--body",
+              "sort-table__row--expanded-area",
+            );
```

Comment:
I wonder if this should mirror `classes` and make a call to `this.props.rowClass(rowIndex)`?

It looks like that prop is only used on this page, to set the job status class to match rules in `styl/pages/jobs.styl`.  Worth looking at the rules that remain.